libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2126,17 +2109,19 @@ bool WaveTrack::Get(size_t iChannel, size_t nBuffers, samplePtr buffers[],
    bool mayThrow, sampleCount * pNumWithinClips) const
 {
    const auto nChannels = NChannels();
-   assert(nBuffers - iChannel <= nChannels); // precondition
+   assert(iChannel + nBuffers <= nChannels); // precondition
```

Comment:
This fix attempt relates to https://github.com/audacity/audacity/pull/4771#discussion_r1239606798

Reply:
You saw my retraction of my mistaken comment of yesterday?

Reply:
I said I finished review, but then I wrote some more in answer to your answers.

Reply:
> You saw my retraction of my mistaken comment of yesterday?

I did, yes, but I still think this assertion is incorrect.